### ISSUE TITLE
[Docs] Guide the user to use openssl to create CSR for X509 auth

### ISSUE THREAD

**Issue by lindarr915** [aws/eks-anywhere#5657]
**What would you like to be added**:

I would like to create one section guiding the user to use X509 cert auth for Kubernetes API auth. 

In other words, the EKS-A version for the docs: 
https://docs.aws.amazon.com/eks/latest/userguide/eks-outposts-network-disconnects.html

Changes are needed for the step 8 and later.  

**Why is this needed**:

The customer would like to have details guides to managed access of EKS-A without external OIDC provider.

**Comment by abhinavmpandey08** [comment]
Hi @lindarr915, thanks for opening the issue.

I'm not sure if we document this on EKS Anywhere or not but if you want to test, you can use the following command to get the cluster CA for EKS Anywhere cluster. This is can be used instead of the first command on step 8
```bash
kubectl get configmap -n kube-system kube-root-ca.crt -ojson | jq -r '.data."ca.crt"'
```

**Comment by lindarr915** [comment]
@abhinavmpandey08 Yes I have tested it and it worked for me. I simply hope it can be documented in EKS Anywhere docs.
